config/urls.py

```diff
@@ -6,7 +6,7 @@
 from scim.api import router as scim_router
 
 
-protected_apis.add_router("scim/v2/Users/", scim_router)
+protected_apis.add_router("scim/v2/Users", scim_router)
 protected_apis.add_router("sso/", sso_profile_router)
```

Comment:
```suggestion
protected_apis.add_router("/scim/v2/Users", scim_router)
protected_apis.add_router("/sso", sso_profile_router)
```
surely we'd want the preceding slash? Either way let's keep it consistent IMHO